pkg/instance/hostname/hostname_test.go

```diff
@@ -0,0 +1,16 @@
+// SPDX-FileCopyrightText: Copyright The Lima Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package hostname
```

Comment:
Why not hostname_test? we don't need to access private names in the package.

Reply:
There are no strong reasons either way, but there are no private names (right now), so it makes no difference, but makes the tests look more cluttered:

```go
	assert.Equal(t, hostname.FromInstName("default"), "lima-default")
	assert.Equal(t, hostname.FromInstName("ubuntu-24.04"), "lima-ubuntu-24-04")
	assert.Equal(t, hostname.FromInstName("foo_bar.baz"), "lima-foo-bar-baz")
```

vs.

```go
	assert.Equal(t, FromInstName("default"), "lima-default")
	assert.Equal(t, FromInstName("ubuntu-24.04"), "lima-ubuntu-24-04")
	assert.Equal(t, FromInstName("foo_bar.baz"), "lima-foo-bar-baz")
```

and I prefer tests to be as simple as possible. But again it doesn't really matter, and I will change it.